### PR TITLE
Control API

### PR DESCRIPTION
**Pull request by veronikaKochugova**

### CODE REVIEW

File: src/main/java/com/emc/mongoose/Main.java [hunk 1]
```diff
@@ -51,124 +39,164 @@
...
+		final Map<String, String> parsedArgs = CliArgUtil.parseArgs(args);
+		final List<Map<String, Object>> aliasingConfig = config.listVal("aliasing");
+		return AliasingUtil.apply(parsedArgs, aliasingConfig);
+	}
+
+	private static void addMetricsService(final Server server) {
+		final ServletContextHandler context = new ServletContextHandler();
+		context.setContextPath("/");
+		server.setHandler(context);
+		//context.addServlet(new ServletHolder(new MetricsServlet()), "/metrics");
```
Comment (akurilov) [inline src/main/java/com/emc/mongoose/Main.java]: is the metrics servlet is disabled by this comment? if yes, why?

File: src/main/java/com/emc/mongoose/Main.java [hunk 2]
```diff
@@ -87,7 +88,7 @@ public static void main(final String... args)
 					runNode(configWithArgs, extensions, metricsMgr);
 				} else {
 					final Server server = new Server(port);
-					addMetricsService(server);
+					addServices(server, defaultConfig);
```
Comment (akurilov) [inline src/main/java/com/emc/mongoose/Main.java]: should be `configWithArgs` instead of `defaultConfig`

Comment (veronikaKochugova) [reply src/main/java/com/emc/mongoose/Main.java]: configWithArgs - it is config with applied CLI args != default

File: src/test/java/com/emc/mongoose/system/ControlAPITest.java [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+/**
+ @author veronika K. on 01.11.18 */
+public class ControlAPITest {
```
Comment (akurilov) [inline src/test/java/com/emc/mongoose/system/ControlAPITest.java]: ControlApiTest

File: src/test/java/com/emc/mongoose/controls/ConfigServletTest.java [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+import java.net.URLConnection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ @author veronika K. on 02.11.18 */
+public class ConfigServletTest {
+
+	private static final int PORT = 9999;
```
Comment (akurilov) [inline src/test/java/com/emc/mongoose/controls/ConfigServletTest.java]: this value should be obtained from the config

Comment (veronikaKochugova) [reply src/test/java/com/emc/mongoose/controls/ConfigServletTest.java]: But it is Unit test, there is no mongoose

Comment (veronikaKochugova) [reply src/test/java/com/emc/mongoose/controls/ConfigServletTest.java]: I can change port in system test (ControlApiTest)

File: build.gradle [hunk 5]
```diff
@@ -53,6 +57,9 @@ allprojects {
 		maven {
 			url "https://oss.sonatype.org/service/local/repositories/releases/content"
 		}
+		maven {
```
Comment (akurilov) [inline build.gradle]: not needed anymore, released the confuse-io-json library

Comment (veronikaKochugova) [reply build.gradle]: Ok

**Comment by veronikaKochugova** [comment]
travic ci was stoped

**Pull request closed without merging**